buildMetrics.sh

```diff
@@ -2,6 +2,8 @@
 # Generates fontMetricsData.js
 PERL="perl"
 PYTHON=`python2 --version >/dev/null 2>&1 && echo python2 || echo python`
+OUTPUT=../../fontMetricsData.js
 
 cd src/metrics
-$PERL ./mapping.pl | $PYTHON ./extract_tfms.py | $PYTHON ./extract_ttfs.py | $PYTHON ./format_json.py --width > ../../fontMetricsData.js
+echo "// This file is GENERATED by buildMetrics.sh. DO NOT MODIFY." >$OUTPUT
```

Comment:
I think this should be in `format_json.py`, where other outputs are.

Reply:
Finally remembered to do this! I confirmed that it generates the same output.